openedx/features/edly/utils.py

```diff
@@ -563,30 +566,77 @@ def compose_certificate_email(student, course, message_context):
         'student_name': student.profile.name,
         'course_name': course.display_name,
         'course_date': datetime.now().strftime('%m/%d/%Y'),
+        'cert_url': '{}{}{}'.format(
+            'https://',
```

Comment:
If `https://` is fixed we can move it to string like `'https://{}{}'.format(var1, var2)`

Reply:
I was trying to make it dynamic but request wasn't available so had to hardcode for now.